dosomething-dev/fastly-backend/main.tf

```diff
@@ -21,10 +21,6 @@ resource "fastly_service_v1" "backends-dev" {
     name = "${var.northstar_domain}"
   }
 
-  domain {
-    name = "${var.phoenix_domain}"
```

Comment:
We'd previously routed `www-dev.dosomething.org` through the backend property, which doesn't make a ton of sense (hence why this is being removed). I'll set up redirects for `www-dev.dosomething.org` and `staging.dosomething.org` to point to `dev.dosomething.org` once this is applied.